libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -1951,6 +1964,13 @@ bool AP_Mission::get_next_cmd(uint16_t start_index, Mission_Command& cmd, bool i
             return false;
         }
 
+        // check for do-jump-tag command and convert to do-jump
+        if (temp_cmd.id == MAV_CMD_DO_JUMP_TAG) {
+            temp_cmd.id = MAV_CMD_DO_JUMP;
+            temp_cmd.content.jump.target = get_index_of_jump_tag(temp_cmd.content.jump.target);
```

Comment:
We should really handle the failed-to-lookup case here.  I know it's supposed to be a "can't-happen"...

Reply:
That's just adding more logic that had the same behavior as a bad do jump